### DISCUSSION TITLE
What is editing-adapter?

### DISCUSSION THREAD

**Discussion by felipe-chamas** [google/model-viewer#2319] [Q&A] [answered]
I was searching through the files and properties, I realized some features are not on Scenegraph, but appears in editing-adapter.
What is this package for? Can I make a PR to update Scenegraph with those options I need?

**Comment by elalish** [comment, marked as answer]
Yes, editing-adapter is legacy and the plan is to remove it once scene-graph has all its functionality. So yes, PRs would be much appreciated! You might try taking over #1979, since that seems to be abandoned.

**Reply by felipe-chamas** [reply]
Sure!! Thank you
